docs/use-cases/exchange.md

```diff
@@ -0,0 +1,136 @@
+---
+id: exchange
+title: Exchange Use Case
+sidebar_label: Exchange
+---
+*By the end of this guide you should know how an exchange can integrate with the OMG Network.*
+
+- [1. Introduction](#1-introduction)
+  * [1.1 Overview](#11-overview)
+  * [1.2 Requirements](#12-requirements)
+    + [User Personas](#user-personas)
+    + [User Stories](#user-stories)
+- [2. Proposed Design](#2-proposed-design)
+  * [2.1 Integration Scope](#21-integration-scope)
+    + [2.1.1 Generate ETH Addresses](#211-generate-eth-addresses)
+    + [2.1.2 Query Blockchain](#212-query-blockchain)
+    + [2.1.3 Make Deposits](#213-make-deposits)
+    + [2.1.4 Make Withdrawals](#214-make-withdrawals)
+    + [2.1.5 Merge UTXO](#215-merge-utxo)
+    + [2.1.6 Setup a Watcher](#216-setup-a-watcher)
+    + [2.1.7 Deposit Funds to the OMG Network](#217-deposit-funds-to-the-omg-network)
+    + [2.1.8 Rebalance Wallets](#218-rebalance-wallets)
+  * [2.2 Integration UI](#22-integration-ui)
+
+## 1. Introduction
+
+### 1.1 Overview
+
+The main goal of this document is to demonstrate how a stablecoin issuer or an exchange can add an OMG Network as one of the options to deposit and withdraw a particular ERC20 token (e.g. USDt).
+
+The integration with the OMG Network can significantly help the entire Ethereum ecosystem to offload a certain volume from the rootchain, as well as introduce faster and cheaper transactions with the same security guarantees as to Ethereum network.
+
+### 1.2 Requirements
+
+#### User Personas
+
+1. Customers - participants and active users/traders of a given exchange.
+2. Integrator - a development and management team within an exchange that evaluates the requirements and delivers the integration with the OMG Network.
```

Comment:
2. Integrators - development and management staff at an exchange that evaluate the requirements and deliver the integration to the OMG Network.